backend/tests/polling_station_integration_test.rs

```diff
@@ -0,0 +1,39 @@
+#![cfg(test)]
+
+use reqwest::StatusCode;
+use sqlx::SqlitePool;
+
+use backend::polling_station::PollingStationListResponse;
+
+use crate::utils::serve_api;
+
+mod utils;
+
+#[sqlx::test(fixtures(path = "../fixtures", scripts("elections", "polling_stations")))]
+async fn test_polling_station_listing_works(pool: SqlitePool) {
```

Comment:
Am not a big fan of "works" in a test name, because it's not very specific. On the other hand, just noticed we do the same in `election_integration_test.rs`.

Reply:
That was indeed my blueprint for this PR. Not sure if I should change it here, or on both locations, or as a separate PR throughout the code?